Sources/SwiftLanguageService/CMakeLists.txt

```diff
@@ -74,7 +74,6 @@ target_link_libraries(SourceKitLSP PRIVATE
   Crypto
   TSCBasic
   PackageModel
```

Comment:
Do we still need PackageModel?

Reply:
Yeah, it's used in a few places. I can look into removing that as well separately!

Reply:
This is going to be addressed by https://github.com/swiftlang/sourcekit-lsp/pull/2245